### DISCUSSION TITLE
loadBalancerIP configuration that doesn't seem to work

### DISCUSSION THREAD

**Discussion by gaoyuan5251** [milvus-io/milvus#38648] [Q&A and General discussion]
背景：helm部署模式，版本v2.4.17，部署环境阿里云ACK pro集群，修改values.yaml文件时遇到的问题，如下。

我想用一下这个LoadBalancer，然后把milvus服务能通过external-ip给代理出去，供其他服务调用。但是我目前指定这个参数以后，我发现，部署的时候，还是会随机给我生成一个公网LB的external-ip，并不是我想指定的内网LB的IP。
<img width="1129" alt="image" src="https://github.com/user-attachments/assets/806a9f9d-27e0-4a69-9941-580c581b15c0" />

**Comment by LoveEachDay** [comment]
If you'd specify a precreated internal lb ip, you can add the following annotations:
```
service.beta.kubernetes.io/alibaba-cloud-loadbalancer-name: "nlb-name"
```

**Comment by gaoyuan5251** [comment]
@LoveEachDay nodePort参数貌似不生效
<img width="1276" alt="image" src="https://github.com/user-attachments/assets/0cf09756-8df6-425c-9499-11d30352ff73" />
还是随机生成：
![image](https://github.com/user-attachments/assets/05d35703-cce6-4309-88ce-de7a3ac50e60)
